src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -945,7 +945,7 @@ def migrate_to_multisite(self, realm_name: str, zonegroup_name: str, zone_name:
     def create_realm(self, realm_name: str, default: bool):
         rgw_realm_create_cmd = ['realm', 'create']
         cmd_create_realm_options = ['--rgw-realm', realm_name]
-        if default != 'false':
+        if default != False:
```

Comment:
Hey @aaSharma14 looks the linter is not liking it

```
lint: commands[0] /home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard> flake8
./services/rgw_client.py:948:20: E712 comparison to False should be 'if cond is not False:' or 'if cond:'
1     E712 comparison to False should be 'if cond is not False:' or 'if cond:'
```
To avoid the linting issue, it should be:
`if default is not False:`"
Or even better:
`if default:`

Reply:
Updated. Thanks @Pegonzal